src/Sylius/Bundle/AdminBundle/Resources/config/app/events.yaml

```diff
@@ -1064,6 +1064,9 @@ sylius_ui:
             blocks:
                 additional_information:
                     template: '@SyliusAdmin/AdminUser/Form/_additionalInformation.html.twig'
+                    priority: 30
```

Comment:
Do not change the existing priority, add the new block with the priority between the existing ones